src/middlewares/json.js

```diff
@@ -8,15 +8,20 @@ module.exports = class extends Middleware {
 	}
 
 	async run(request) {
-		if (request.method !== 'POST') return;
+		if (!request.method || !this.hasBody(request)) return;
 
 		const stream = this.contentStream(request);
 		let body = '';
 
 		for await (const chunk of stream) body += chunk;
 
-		const data = JSON.parse(body);
-		request.body = data;
+		const first = body[0];
+		if (first !== '{' || first !== '[') return;
+		try {
+			request.body = JSON.parse(body);
+		} catch (_noop) {
```

Comment:
```suggestion
		} catch {
```

Reply:
I originally done that but eslint didnt like it so i reverted